### PR TITLE
Add publish my addon block

### LINKED ISSUE #4208: [DevHub Landing] Publish your add-on content block
Looks like this in the mocks

d94a454a-aca8-11e6-852f-92284eda5db0_png__png_image__1280 x 2850_pixels_

710ffb0e-aca9-11e6-8eeb-6a75df8e0da2_png__png_image__320 x 2860_pixels_

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #4208

Animation showing breakpoint changes:

### CODE REVIEW

File: static/css/devhub/new-landing/base.less [hunk 1]
```diff
@@ -85,5 +96,3 @@ h3 {
 // Content sections
 @import 'sections/overview';
 @import 'sections/more-information';
-@import 'sections/my-addons';
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/base.less]: Why remove it?

File: static/css/devhub/new-landing/common.less [hunk 2]
```diff
@@ -60,3 +64,91 @@
...
 }
+
+.DevHub-content-container {
+  display: flex;
+  position: relative;
+}
+
+.DevHub-content-header {
+  font-size: 40px;
+  font-weight: 500px;
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: the px again :)

File: static/css/devhub/new-landing/base.less (hunk 1 above)
Comment (muffinresearch) [inline static/css/devhub/new-landing/base.less]: Good catch, that's a mis-cue on my part.

File: static/css/devhub/new-landing/base.less [hunk 3]
```diff
@@ -3,8 +3,9 @@
 
 // Grid
 @grid-max: 940px;
-@small: ~"only screen and (max-width: 599px)";
-@medium: ~"only screen and (min-width: 600px)";
+@extra-small: ~"only screen and (max-width: 599px)";
+@small: ~"only screen and (min-width: 599px)";
```
Comment (muffinresearch) [inline static/css/devhub/new-landing/base.less]: This puts small/medium back to what they were when I started but I've added a extra-small for a bit more granularity.

File: static/css/devhub/new-landing/common.less [hunk 4]
```diff
@@ -60,3 +64,91 @@
...
+  max-width: 66%;
+}
+
+.DevHub-content-image-wrapper {
+  position: absolute;
```
Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: I'll fix up this sorting too. 🍟

**Comment by EnTeQuAk** [comment]
oh absolutely! No, I think the comments I had were everything I had.

**Pull request merged**